stake-token-prices/README.md

```diff
@@ -0,0 +1,57 @@
+# Stake Token Price Service
+
+This service periodically collects token prices (in USD) from CoinGecko or CoinMarketCap, stores those in a key/value store and exposes an HTTP endpoint to easily get the stored prices.
+
+## Components
+
+| Component | Description                                               |
+| --------- | --------------------------------------------------------- |
+| `cache`   | Redis database to store the token prices                  |
+| `cron`    | Node cron job that pulls the prices and updates the cache |
+| `api`     | Node HTTP server to retrieve the prices from the cache    |
+
+This architecture allows making very few calls to the price APIs, scales well with the user's base and even allows the prices to be fed even if the pricing APIs are temporarily down.
+
+### Configuration
+
+These environment variables control how the `cron` job behaves:
+
+| Variable                                     | Description                                                                         | Default               |
+| -------------------------------------------- | ----------------------------------------------------------------------------------- | --------------------- |
+| CACHE_EXPIRATION_SEC                         | How long the prices will be kept in the cache.                                      | 3600                  |
+| REFRESH_INTERVAL_SEC                         | How frequently the cache will be refreshed. If set to 0, it will run once and exit. | 300                   |
```

Comment:
`3600` and `300` - What units are these? I assume `ms`, but it also could be seconds 🤔

Reply:
so stupid, I just noticed the `_SEC` suffix 🤦🏽